backend/routes/parking.py

```
@@ -0,0 +1,140 @@
+
+    # Validate file
+    if not file.content_type or not file.content_type.startswith('image/'):
+        raise InvalidImageError("File must be an image")
+    
+    try:
+        # Read image data
+        image_bytes = await file.read()
+
+        # Validate size (10MB limit)
+        if len(image_bytes) > 10 * 1024 * 1024:
+            raise InvalidImageError("Image file too large (max 10MB)")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)